### PR TITLE
EZP-29635: Styled labels in Admin Panel Edit views

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-29635](https://jira.ez.no/browse/EZP-29635)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

**Most important aspects:**
- Harmonized form labels in Editing views by styling existing class labels when permitted or by adding a new specific class `ez-label`, as well as adding a specific class for the view, `ez-admin-edit`;
- **Need help for exceptional cases:**
    -> Content type specific Edit view for field type `ezdatetime`: Year, Month, Day, Minute, Second;
    -> Roles specific Edit view `limitation` group.

ez platform 1
ez platform 2
ez platform 4
ez platform 6
ez platform 7
ez platform 8
ez platform 10
ez platform 11
ez platform 12
ez platform 13
ez platform 14
ez platform 15

**Would need help _>**
    -> Content type specific Edit view for field type `ezdatetime`: Year, Month, Day, Minute, Second;
    -> Roles specific Edit view `limitation` group.

ez platform 3
ez platform 5

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_card.scss [hunk 1]
```diff
@@ -60,6 +69,73 @@
     }
 }
 
+.ez-admin-edit {
+    .ez-card {
+        .form-group,
+        .ez-card--fieldtype-container {
+            .ez-label {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_card.scss]: What do you think about making it a generic class. I mean not nested inside another classes.
Just something like this:
```scss
.ez-label {
    margin-bottom: 0;
    font-weight: 700;
    &:after {
        content: ': ';
    }
}
```
They same rule I would apply for the `.required` class.

Comment (inakijv) [reply src/bundle/Resources/public/scss/_card.scss]: It would be great (I considered that option too). It will simplify more the process.

**Comment by inakijv** [comment]
Updated based on review.

Still facing issues here:
-> Content type specific Edit view for field type ezdatetime: Year, Month, Day, Minute, Second;
-> Roles specific Edit view limitation group.

**Comment by sunpietro** [comment]
@inakijv what are these issues you mentioned?

**Comment by inakijv** [comment]
@sunpietro two views still need styling (I need suggestions/help on how to get them):
- Admin panel / Content type Group / Content Type specific /Edit view (also Create view too) -> Field type ezdatetime, the labels: Year, Month, Day, Minute, Second;
- Admin panel / Roles / Roles specific / Edit view -> labels for limitation group.

File: src/bundle/Resources/public/scss/_card.scss [hunk 2]
```diff
@@ -60,6 +60,28 @@
     }
 }
 
+.ez-label {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_card.scss]: If this is a separate component it should be in different file `_label.scss`.

Comment (inakijv) [reply src/bundle/Resources/public/scss/_card.scss]: Ok.

**Comment by inakijv** [comment]
Updated based on review.

File: src/bundle/Resources/public/scss/_labels.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.ez-label {
+    margin-bottom: 0;
+    font-weight: 700;
+
+    &:after {
+        content: ': ';
+    }
+}
+
+.ez-label.checkbox-inline,
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_labels.scss]: It should go inside the `.ez-label {` declaration like this: `&.checkbox-inline`

File: src/bundle/Resources/public/scss/_labels.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+.ez-label.form-check-label {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_labels.scss]: It should go inside the `.ez-label {` declaration like this: `&.form-check-label`

File: src/bundle/Resources/public/scss/_labels.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    margin-bottom: .0625rem;
+
+    &:after {
+        content: '';
+    }
+}
+
+.ez-label.form-check-label:not(.checkbox-inline) {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_labels.scss]: If you follow my suggestions above, then it should be placed inside the `&.form-check-label {` selector, like this: `&:not(.checkbox-inline)`

File: src/bundle/Resources/public/scss/_labels.scss (hunk 3 above)
Comment (inakijv) [reply src/bundle/Resources/public/scss/_labels.scss]: Ok.

File: src/bundle/Resources/public/scss/_labels.scss (hunk 4 above)
Comment (inakijv) [reply src/bundle/Resources/public/scss/_labels.scss]: Ok.

File: src/bundle/Resources/public/scss/_labels.scss (hunk 5 above)
Comment (inakijv) [reply src/bundle/Resources/public/scss/_labels.scss]: Perfect. Thanks.

**Comment by inakijv** [comment]
Updated based on review.

**Comment by inakijv** [comment]
Updated based on review.

ez platform 27

**Pull request merged**